### PR TITLE
Add vergunningen layer and updated documentation

### PR DESCRIPTION
**Pull request by chrisvanmook**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,3 +15,76 @@ the Elastic Search from acceptance to localhost with :
...
+  }
+}
+```
+
+- _id_ - this is the ID you just added to one of the collections in [map-collections.config.json](./assets/map-collections.config.json)
+- _layers_ - an array of the layers available in the `.map` file. These `.map` files can be found in [this repo](https://github.com/Amsterdam/mapserver/)
+- _title_ - this is the title that will be displayed in the front-end.
+- _type_ - usually just `wms`
+- _url_ - this is the url to retrieve the actual (wms) layer to be displayed on the map, but it's also used to retrieve the legend images (which are specified in the `legendItems` field)
+- _legendItems_ - an array of objects containing a `title`, that will be displayed in the frontend but can also be used to retrieve the image for the legens from the mapserver (checkout the corresponding `.map` file in [this repo](https://github.com/Amsterdam/mapserver/))
```
Comment (royderks) [inline README.md:59]: Mention the `imageRule` here? 

The field `title` is used by the mapserver to create an icon for this layer or legend item. If the title doesn't match the title used in the `.map` file (for redactional purposes), the `imageRule` field can be used to define the legend icon.

File: assets/map-layers.config.json [hunk 2]
```diff
@@ -4696,7 +4696,7 @@
         "title": "Onbekend"
       },
       {
-        "imageRule": "< 1800",
+        "description": "< 1800",
```
Comment (royderks) [inline assets/map-layers.config.json:4699]: Why is this changed?

Comment (chrisvanmook) [reply assets/map-layers.config.json:4699]: oops my bad

File: README.md (hunk 1 above)
Comment (chrisvanmook) [reply README.md:59]: Will change description 👍  imageRule is mentioned below btw

Comment (royderks) [reply README.md:59]: Ah overlooked that one :)
